src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -27,10 +27,8 @@
 #if CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE
 
 #include <platform/internal/BLEManager.h>
+#define RSI_BLE_ENABLE 1
```

Comment:
Is this define really needed ? It doesn't seems to be used in the changes you've done...